### DISCUSSION TITLE
Why support for `SystemDecorations="BorderOnly"` was deprecated in Linux

### DISCUSSION THREAD

**Discussion by SeWZC** [AvaloniaUI/Avalonia#16215] [Q&A]
OS: Ubuntu 22.04.4 LTS
desktop: GNOME 42.9

When SystemDecorations="Full":
![image](https://github.com/AvaloniaUI/Avalonia/assets/36623158/b4cc4a07-428d-42be-86e1-ed9ec81b4fc2)

When SystemDecorations="None" or SystemDecorations="BorderOnly":
![image](https://github.com/AvaloniaUI/Avalonia/assets/36623158/c4551162-544b-4389-b78c-db92ed2e6af7)

Currently, "None" is the same as "BorderOnly" for SystemDecorations on Linux. See #3849.
The x11window's MotifWmHints property is 0x3, 0x3e, 0x0, 0x0, 0x0 if the window's SystemDecorations is set to None or BorderOnly.
If you use `xprop -f _MOTIF_WM_HINTS 32x -set _MOTIF_WM_HINTS "0x3,0x3e,0x2,0x0,0x0"` to change the MotifWmHints property, it will result in a true BorderOnly. Look at the shadow:
![image](https://github.com/AvaloniaUI/Avalonia/assets/36623158/3baa8d58-320d-4104-8931-e8d28f2f35df)

I‘m a beginner with Avalonia and Linux. I want to know why support for SystemDecorations="BorderOnly" was deprecated on Linux.

**Comment by rabbitism** [comment]
Which avalonia version are you testing on?

**Reply by SeWZC** [reply]
I'm sorry. It's 11.0.10.
